binr/radiff2/radiff2.c

```diff
@@ -166,6 +166,7 @@ static int show_help(int v) {
 		"  -r         output in radare commands\n"
 		"  -s         compute text distance\n"
 		"  -ss        compute text distance (using levenstein algorithm)\n"
+		"  -S  [name] sort code diff\n"
```

Comment:
change description to "sort code diff by column [name, address, type, size, dist]" <- enumerate them all